### ISSUE TITLE
update to google_api_headers: ^1.0.0

### ISSUE THREAD

**Issue by sgehrman** [fluttercommunity/flutter_google_places#150]
google_api_headers <0.2.0-nullsafety.0 depends on package_info >=0.4.0 <2.0.0 and google_api_headers >=0.2.0-nullsafety.0 <1.0.0 depends on package_info ^0.5.0-nullsafety, google_api_headers <1.0.0 requires package_info >=0.4.0 <2.0.0.
And because flutter_google_places >=0.2.7 depends on google_api_headers >=0.1.0 <1.0.0, flutter_google_places >=0.2.7 requires package_info >=0.4.0 <2.0.0

**Comment by hoc081098** [comment]
Workaround:
```yaml
dependency_overrides:
  flutter_google_places:
    git:
      url: https://github.com/hoc081098/flutter_google_places.git
      ref: 0566867ee2abe5c8c2ba180678f18309845b23a6
```

**Comment by calistus** [comment]
Nice work with the upgrade.@hoc081098 
You saved my time

**Comment by erickhaendel** [comment]
> Workaround https://github.com/hoc081098/flutter_google_places:
> 
> ```yaml
> dependency_overrides:
>   flutter_google_places:
>     git:
>       url: https://github.com/hoc081098/flutter_google_places.git
>       ref: 7b6c2aa2053131142c0e48639823d24f312d00c1
> ```

Nice!

**Comment by KevMorelli** [comment]
Implemented on v0.3.0

**Comment by mmkkpp666111** [comment]
> google_api_headers <0.2.0-nullsafety.0 depends on package_info >=0.4.0 <2.0.0 and google_api_headers >=0.2.0-nullsafety.0 <1.0.0 depends on package_info ^0.5.0-nullsafety, google_api_headers <1.0.0 requires package_info >=0.4.0 <2.0.0.
> And because flutter_google_places >=0.2.7 depends on google_api_headers >=0.1.0 <1.0.0, flutter_google_places >=0.2.7 requires package_info >=0.4.0 <2.0.0

Dbdf
